src/core/qgsfeaturerequest.h

```diff
@@ -100,6 +100,16 @@ class CORE_EXPORT QgsFeatureRequest
       GeometryAbortOnInvalid = 2, //!< Close iterator on encountering any features with invalid geometry. This requires a slow geometry validity check for every feature.
     };
 
+    /**
+     * Filter handling behavior
+     */
+    enum FilterCombination
+    {
+      UseLast        = 0,   //!current behavior
+      OperatorAND    = 1,   //! stack filters
+      OperatorOR     = 999  //! OR operator, not implemented
```

Comment:
```suggestion
#if 0
      OperatorOR     = 999  // TODO
#endif
```

Reply:
Sip throws a syntax error when building, not sure why, but the layout test is now ok with the file.

Reply:
ended up using another less optimal syntax to make the tests pass.